### PR TITLE
Add CertificateRequests dashboard

### PR DESCRIPTION
**Pull request by mcharriere**
This PR

- Adds CertificateRequests dashboard

### Checklist

- [x] Update changelog in CHANGELOG.md in an end-user friendly language.

### CODE REVIEW

File: helm/dashboards/dashboards/shared/private/certificate-requests.json [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+            "uid": "PBFA97CFB590B2093"
+          },
+          "editorMode": "code",
+          "expr": "sum(etcd_kubernetes_resources_count{cluster_id=~\"$cluster_id\", kind=\"certificates.cert-manager.io\"}) by (cluster_id)",
+          "legendFormat": "__auto",
+          "range": true,
+          "refId": "A"
+        }
+      ],
+      "title": "Certificates per WC",
```
Comment (TheoBrigitte) [inline helm/dashboards/dashboards/shared/private/certificate-requests.json:115]: I think this request show certificates for all clusters MC and WC, and its a good thing. Let's avoid being too specific.

```suggestion
      "title": "Certificates per cluster",
```

File: helm/dashboards/dashboards/shared/private/certificate-requests.json [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+            "uid": "PBFA97CFB590B2093"
+          },
+          "editorMode": "code",
+          "expr": "sum(etcd_kubernetes_resources_count{cluster_id=~\"$cluster_id\", kind=\"certificaterequests.cert-manager.io\"}) by (cluster_id)",
+          "legendFormat": "__auto",
+          "range": true,
+          "refId": "A"
+        }
+      ],
+      "title": "CertificateRequests per WC",
```
Comment (TheoBrigitte) [inline helm/dashboards/dashboards/shared/private/certificate-requests.json:202]: same remark

```suggestion
      "title": "CertificateRequests per cluster",
```

**Review by TheoBrigitte — APPROVED** [review]
LGTM

But I would appreciate a screenshot of the dashboard, just to have a feeling for it.

File: helm/dashboards/dashboards/shared/private/certificate-requests.json (hunk 1 above)
Comment (mcharriere) [reply helm/dashboards/dashboards/shared/private/certificate-requests.json]: you're right. fixed.

**Comment by mcharriere** [comment]
image

**Pull request merged**